### DISCUSSION TITLE
Recaptcha V3 ne fonctionne pas

### DISCUSSION THREAD

**Discussion by marko78700** [AmauriC/tarteaucitron.js#913] [Q&A]
Bonjour je viens d'installer tarteaucitron sur un site où j'utilise ReCaptcha V3.

Depuis l'intégration de TAC, la validation de mon formulaire ne fonctionne pas
Dans mon code en backend, l'endroit où je suis censé récupérer mon token est toujours vide.

Aussi, j'ai le message suivant dans la console du navigateur 

grecaptcha is not defined

Je suis un peu perdu :-(

Merci d'avance pour votre aide

**Comment by AmauriC** [comment]
Si `grecaptcha is not defined` est dans la console ça indique que le site essai d'accéder à reCaptcha avant qu'il ne soit chargé par tarteaucitron.

Avec l'url du site je pourrai, peut-être, trouver une solution ; )
